Description: Disable nose doctest as requires network access
Author: Olivier Berger <obergix@debian.org>
Forwarded: not-needed
--- a/pyproject.toml
+++ b/pyproject.toml
@@ -151,7 +151,6 @@
 
 [tool.pytest.ini_options]
 addopts = [
-   "--doctest-modules",
    "--ignore=admin",
    "--ignore=devtools",
    "--ignore=rdflib/extras/external_graph_libs.py",
@@ -159,7 +158,6 @@
    "--doctest-glob=docs/*.rst",
    "--strict-markers",
 ]
-doctest_optionflags = "ALLOW_UNICODE"
 filterwarnings = [
     # The below warning is a consequence of how pytest doctest detects mocks and how DefinedNamespace behaves when an undefined attribute is being accessed.
     "ignore:Code. pytest_mock_example_attribute_that_shouldnt_exist is not defined in namespace .*:UserWarning",
